docs/configuration/layers/esri-dynamic.md

```diff
@@ -61,9 +61,11 @@ The URL for the DataBC Layer Catalog.
 ## DynamicLayers Property
 `"dynamicLayers": Array`
 
-A listing of dynamic layer configurations.
-This will typically only contain one dynamic feature, which can be derived from the MPCM Layer Catalog.
-
+A listing of dynamic layer configurations in JSON format. This will typically only contain one dynamic feature. This configuration comes from the MPCM Layer Catalog.
```

Comment:
I know it already existed, but 'dynamic feature' doesn't make sense. I'd suggest something like 'This will typically contain configuration for a single layer. The default configuration comes from the MPCM Layer Catalog.'

Reply:
I'm fine with your suggestion. Done.